data/upload_index_data.py

```diff
@@ -1,10 +1,10 @@
 import os
+import json
 import openai
-import pandas as pd
 
 from dotenv import load_dotenv
 from azure.core.credentials import AzureKeyCredential
-from azure.search.documents import SearchClient
+from azure.search.documents import SearchIndexingBufferedSender
```

Comment:
made document, why adding this change, I think it has to do with the amount of data you are now loading but maybe it would be good to have comment about it.